api/controllers/admin.js

```diff
@@ -11,7 +12,8 @@ module.exports = {
 };
 
 function config(req, res) {
-  res.json(req.app.get("config"));
+  var redact_passwords = Scrubbers.object_keys(["password"]);
```

Comment:
Couple of super minor nit questions:

1) This is probably my ignorance wrt style, but why caps for Scrubbers? I thought caps were more common when the object was a class prototype, but you're just using Scrubbers as an object. 

2) Why create redact_passwords on every call? Can't it just be cached up with the requires?